Sources/Logging/Logging.swift

```diff
@@ -498,8 +498,16 @@ public struct MultiplexLogHandler: LogHandler {
     }
 }
 
+private struct StandardErrorOutputStream: TextOutputStream {
+    func write(_ string: String) {
+        string.withCString { ptr in
+            _ = fputs(ptr, stderr)
+        }
+    }
+}
+
 /// Ships with the logging module, really boring just prints something using the `print` function
-internal struct StdoutLogHandler: LogHandler {
+internal struct StderrLogHandler: LogHandler {
```

Comment:
=> i suggest we take an enum here to allow the user the choose between stdout and stderr

=> as for default, its good question!

Reply:
@tomerd sure, could do that. `StdioLogHandler(stream: StdioLogHandler.Stream)` with `enum Stream { case stdout; case stderr }`

Reply:
I think I like it. Default: dunno :)

Reply:
lets keep the default stdout, CLIs can fairly easily switch to stderr with this design